app/views/images/_complete.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="card prx-uploads">
+  <div class="card-header-info d-flex align-items-center">
+    <h5 class="card-title flex-grow-1"><%= t("images.title.#{image.model_name.singular}") %></h5>
```

Comment:
All episode uploaders are cards.  And we share the markup for their various states.

But because the content of the card-header changes based on the image status (showing/hiding the info and destroy icons) ... I had to put the entire card inside the turbo-frame.

Just FYI, why there's a bit of repeated code!